plugins/testsuite/hashing_test.sma

```diff
@@ -0,0 +1,53 @@
+// vim: set ts=4 sw=4 tw=99 noet:
+//
+// AMX Mod X, based on AMX Mod by Aleksander Naszko ("OLO").
+// Copyright (C) The AMX Mod X Development Team.
+//
+// This software is licensed under the GNU General Public License, version 3 or higher.
+// Additional exceptions apply. For full license details, see LICENSE.txt or visit:
+//     https://alliedmods.net/amxmodx-license
+
```

Comment:
This file should be also inserted in the list in `support/AMBuildScript`

I would also do some changes:
* Use a server commands
* Don't use `static`
* No need to initialize
* Output wtih `server_print` or `log_amx`